MIT.md

```diff
@@ -0,0 +1,10 @@
+## Copyright 2021, [Jean BULAMBO RIZIKI]
```

Comment:
You did not change the year as requested by the previous reviewer, please do so.